src/cli/README.md

```diff
@@ -1383,10 +1383,10 @@ Print table of routers.
 
 ```bash
 > router table
-| ID | RLOC16 | Next Hop | Path Cost | LQI In | LQI Out | Age | Extended MAC     |
-+----+--------+----------+-----------+--------+---------+-----+------------------+
-| 21 | 0x5400 |       21 |         0 |      3 |       3 |   5 | d28d7f875888fccb |
-| 56 | 0xe000 |       56 |         0 |      0 |       0 | 182 | f2d92a82c8d8fe43 |
+| ID | RLOC16 | Next Hop | Path Cost | Link Quality In | Link Quality Out | Age | Extended MAC     |
++----+--------+----------+-----------+-----------------+------------------+-----+------------------+
+| 21 | 0x5400 |       21 |         0 |               3 |                3 |   5 | d28d7f875888fccb |
+| 56 | 0xe000 |       56 |         0 |               0 |                0 | 182 | f2d92a82c8d8fe43 |
```

Comment:
I have made the output shorter, the current length is 86 columns, which is the same with the [scan result](https://github.com/openthread/openthread/blob/master/src/cli/README.md#scan-channel).